test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -106,5 +106,38 @@ public class AccessLanguageHeaderRequestCultureProviderTest
                 var response = await client.GetAsync(string.Empty);
             }
         }
+        
+        [Fact]
+        public async void OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture()
```

Comment:
Oops .. sorry, seems introducing ```SupportedUICultures``` property cause the error
```csharp
SupportedUICultures = new List<CultureInfo>
{
    new CultureInfo("ar-YE")
}
```
Can you check please ..